includes/functions/functions_products.php

```diff
@@ -1043,7 +1043,7 @@ function zen_remove_product($product_id, $ptc = 'true')
                   WHERE products_id = " . (int)$product_id);
 
     $db->Execute("DELETE FROM " . TABLE_CUSTOMERS_BASKET_ATTRIBUTES . "
-                  WHERE products_id = " . (int)$product_id);
+                  WHERE products_id LIKE " . (int)$product_id) . ':%';
```

Comment:
Isn't this line missing the opening and closing `'`'s:
```
   WHERE products_id LIKE '" . (int)$products_id . ":%'";
```

Reply:
@brittainmark Would you please fix? This is otherwise ready to be merged.